src/dropdown/dropdown.scss

```diff
@@ -287,6 +288,8 @@ $height-small: 30px !default;
 
       &.is-small {
         .Select-control .Select-multi-value-wrapper {
+          min-height: auto;
```

Comment:
I think it is to beat the `.is-small .Select-multi-value-wrapper { min-height: $height-small; }`